h/services/annotation_json.py

```diff
@@ -71,6 +76,10 @@ def present(self, annotation: Annotation):
                 "target": annotation.target,
                 "document": DocumentJSONPresenter(annotation.document).asdict(),
                 "links": self._links_service.get_all(annotation),
+                "mentions": [
+                    MentionJSONPresenter(mention).asdict()
+                    for mention in annotation.mentions
+                ],
```

Comment:
We should probably return this only if the `at_mentions` feature flag is enabled.